src/header/Header.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './header.css'
+
+function header() {
+  return (
+    <header>
+      <div className="header-logo">
+        <img src={`./images/logo.png`} className="header-image" alt="logo" />
```

Comment:
I wonder if this works?
```suggestion
        <img src="./images/logo.png" className="header-image" alt="logo" />
```
Or:
```
import logo from './images/logo.png';
```
then
```suggestion
        <img src={logo} className="header-image" alt="logo" />
```

Reply:
Hello @raybesiga 

Thanks for the suggestions. They're all really good suggestions.
1. **View each location on a map.** This is a wonderful idea and I'll definitely implement it. 

The other two suggestions are equally great. But they were not part of the description as per the design. Kindly look at the Figma design ( https://www.figma.com/file/YhtX4VSXpgUB4y1HnAeqBq/Travel-Journal-(Copy)?node-id=0%3A1 ). I know it's wonderful to sometimes go above and beyond but time is also a limited resource. I was suggesting I focus on what the tutorial requires at the moment, just to be able to move quickly, and start contributing earlier. 

The goal on this particular project was to **learn** how to pick data from an API.

Reply:
Hello @KnightWorker37 

Your suggestion is really good. My images are currently stored in the `public` folder and I can't access them from the `src` folder, so that was the only alternative. The instructor says though that He will teach how to keep all images in the `src` folder in a later tutorial.